js/buda.js

```diff
@@ -558,21 +558,22 @@ module.exports = class buda extends Exchange {
         //
         const id = this.safeString (order, 'id');
         const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
+        const datetime = this.iso8601 (timestamp);
         const marketId = this.safeString (order, 'market_id');
         const symbol = this.safeSymbol (marketId, market, '-');
         const type = this.safeString (order, 'price_type');
         const side = this.safeStringLower (order, 'type');
         const status = this.parseOrderStatus (this.safeString (order, 'state'));
         const originalAmount = this.safeValue (order, 'original_amount', []);
-        const amount = this.safeNumber (originalAmount, 0);
+        const amount = this.safeString (originalAmount, 0);
         const remainingAmount = this.safeValue (order, 'amount', []);
-        const remaining = this.safeNumber (remainingAmount, 0);
+        const remaining = this.safeString (remainingAmount, 0);
         const tradedAmount = this.safeValue (order, 'traded_amount', []);
-        const filled = this.safeNumber (tradedAmount, 0);
+        const filled = this.safeString (tradedAmount, 0);
         const totalExchanged = this.safeValue (order, 'totalExchanged', []);
-        const cost = this.safeNumber (totalExchanged, 0);
-        const limitPrice = this.safeValue (order, 'limit', []);
-        let price = this.safeNumber (limitPrice, 0);
+        const cost = this.safeString (totalExchanged, 0);
+        const limitPrice = this.safeString (order, 'limit', []);
```

Comment:
The `limitPrice` here is an array (note the default fallback-value of `[]`), we take the 0th element from that array on the next line, so this line must remain `safeValue` and by default it will return an array if 'limit' is missing in the orer structure.